CMake/options.cmake

```diff
@@ -651,7 +651,8 @@ if(FLTK_BUILD_GL)
     unset(HAVE_GL_GLU_H CACHE)
     find_file(HAVE_GL_GLU_H GL/glu.h PATHS ${X11_INCLUDE_DIR})
   else()
-    find_package(OpenGL)
+    find_package(OpenGL COMPONENTS OpenGL)
+    set(OPENGL_LIBRARIES -lGLU -lOpenGL)
```

Comment:
This seems wrong. Adding explicit linker options `-lGLU -lOpenGL` unconditionally w/o even checking the result of `find_package(OpenGL COMPONENTS OpenGL)` doesn't seem correct. Those linker options should be taken from the result of `find_package()` (targets) and not by directly specifying them (even if this is done elsewhere in our CMake files).